ynput/OpenPype
# fix the bug of fbx loaders in Max

## Changelog Description
bug fix of fbx loaders for not being able to parent to the CON instances while importing cameras(and models) which is published from other DCCs such as Maya.

## Additional info
n/a

## Testing notes:
1. Launch Max via Openpype
2. Click "Load...."
3. Load Camera/Model Instance which is published from other DCCs such as maya
4. Also try to load Camera/Model Instance which is published within 3dsMax.
5. Both should free with error shown in the log in 3dsMax
